lib/modulesync.rb

```diff
@@ -11,10 +10,7 @@
 require 'monkey_patches'
 
 GITHUB_TOKEN = ENV.fetch('GITHUB_TOKEN', '')
-
-Octokit.configure do |c|
-  c.api_endpoint = ENV.fetch('GITHUB_BASE_URL', 'https://api.github.com')
-end
+GITLAB_TOKEN = ENV.fetch('GITLAB_TOKEN', '')
```

Comment:
I think we can avoid these constants and only use them inline since they're only used in a single place.

Reply:
Good point, totally missed it. Will adjust accordingly